.github/workflows/production.yml

```
@@ -25,7 +25,9 @@ jobs:
         run: ./gradlew assemble -Pdependency.keycloak.version=${{ secrets.KEYCLOAK_VERSION }}
 
       - name: Build Docker image
-        run: docker build --build-arg keycloak_version=${{ secrets.KEYCLOAK_VERSION }} --platform=linux/amd64 -t revdojo-keycloak .
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:2:188: Double quote to prevent globbing and word splitting

(shellcheck)